src/elements/badge/badge.css

```diff
@@ -0,0 +1,15 @@
+.badge.contained {
+  color: var(--white);
+  background: var(--primary);
+}
+
+.badge {
+  padding: 2px 4px;
+  font-size: .625rem;
+  line-height: .75rem;
+  color: var(--primary);
+  text-transform: uppercase;
+  letter-spacing: .015em;
+  border-radius: 4px;
+  box-shadow: var(--elevation-01);
```

Comment:
@viktor-yakubiv I added elevation here even it is not in Figma. It looks better to me when the background is white. What do you think? Leave it or remove it?

Reply:
Elevation increases the visibility of the element but also it makes the element seem to be interactive. In this case, we want neither of these features.